htdocs/langs/en_US/main.lang

```diff
@@ -773,6 +773,7 @@ ByMonth=By month
 ByDay=By day
 BySalesRepresentative=By sales representative
 LinkedToSpecificUsers=Linked to a particular user contact
+LinkedToSpecificThirdParty=Linked to a particular thirdparty contact
```

Comment:
There is several pb with that:

1) You should use instead a label of combo that is
LinkedToAThirdPartyContactThatHasTheType=Linked to at least one thirdparty contact that has type ...

2) If contract is linked to 2 contacts C1 and C2 with type XXX, the number of lines in list will be duplicated.

3) It is a very strange need, so you should introduce a hidden option for that such a strange feature should not be provided by default to end users.